packages/soci-snapshotter/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "soci-snapshotter"
+version = "0.5.0"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
+
+[package.metadata.build-package]
+releases-url = "https://github.com/awslabs/soci-snapshotter/releases"
+
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/larvacea/soci-snapshotter/archive/refs/tags/v0.5.0.tar.gz"
+sha512 = "9980688b71c149ea0f36c52906ba0cc862ca65b9ee0e4b813fef0df83716e313137886986cc78b694e35516fb21095b0cd0436caf485d4b63bd01b150769b6f4"
+bundle-root-path = "soci-snapshotter-0.5.0/cmd"
+bundle-output-path = "bundled-cmd.tar.gz"
+bundle-modules = [ "go" ]
+
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/larvacea/soci-snapshotter/archive/refs/tags/v0.5.0.tar.gz"
+sha512 = "9980688b71c149ea0f36c52906ba0cc862ca65b9ee0e4b813fef0df83716e313137886986cc78b694e35516fb21095b0cd0436caf485d4b63bd01b150769b6f4"
+bundle-modules = [ "go" ]
+
+# RPM BuildRequires
+[build-dependencies]
+glibc = { path = "../glibc" }
+libz = { path = "../libz" }
+[dependencies]
```

Comment:
New line here to be consistent with other packages, see [this]

[this]: https://github.com/bottlerocket-os/bottlerocket/blob/507c21fa421f073861480be967ea8d053561e18b/packages/containerd/Cargo.toml#L22